internal/elasticsearch/ingest/datastream.go

```diff
@@ -82,7 +82,13 @@ func loadIngestPipelineFiles(dataStreamPath string, nonce int64) ([]Pipeline, er
 		pipelineFiles = append(pipelineFiles, files...)
 	}
 
-	linksFS, err := files.CreateLinksFSFromPath(elasticsearchPath)
+	root, err := files.FindRepositoryRoot()
+	if err != nil {
+		return nil, fmt.Errorf("finding repository root failed: %w", err)
+	}
+	defer root.Close()
+
+	linksFS, err := files.CreateLinksFSFromPath(root, elasticsearchPath)
```

Comment:
@mrodm @jsoriano I am not quite sure how to trigger this code, i see is part of benchmarking tests? but cant seem to make it run ... 🤔  what am i missing?